bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -33,12 +33,6 @@
 
 {% block body_class %}sky{% endblock %}
 
-{% block optimizely %}
```

Comment:
When Optimizely tests are finished we generally just turn the waffle switch off again. Removing the block means we have to add it back again next time we want to run another test, which requires another code change and prod push.